neurom/stats.py

```diff
@@ -48,3 +51,26 @@ def fit(data, distribution='norm'):
     '''
     params = getattr(_st, distribution).fit(data)
     return params, _st.kstest(data, distribution, params)
+
+
+def optimal_distribution(data, distr_to_check=('norm', 'expon', 'uniform')):
```

Comment:
I have to figure out if there are any potential problems with this default parameter, but if there are we can fix it later. So +2 from me. Shall I merge?